app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -365,7 +365,11 @@ def flavor(request)
   end
 
   def number_of_cpus(prov, cloud, flavor_obj)
-    return flavor_obj.try(:cpus) if cloud
+    if cloud
```

Comment:
I'm guessing this block can be condensed into some cool Ruby one-liner :grin:

Reply:
Not one-line but maybe :point_down: 
```
num_cpus = flavor_obj.try(:cpus) if cloud
return num_cpus if num_cpus.present?
```